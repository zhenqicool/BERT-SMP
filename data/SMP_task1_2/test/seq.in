合 肥 明 天 的 湿 度 如 何
摩 尔 勇 士
把 我 打 开 淘 宝 的 网 页
上 山 居 网
赶 集 网
四 川 新 闻 网
苏 宁 易 购
把 淘 宝 网 打 开
看 网 易 财 经
我 想 看 一 下 凤 凰 网
打 开 河 南 英 东 网 站
激 动
新 蛋 网
中 华 军 事 网
打 开 苹 果 香 港 官 方 网 站
索 尼 爱 立 信 官 网
打 开 三 星 首 页
打 开 4 3 9 9 小 游 戏
打 开 宽 带 商 务 网 站
现 在 我 也 想 上 新 浪 网 了
给 我 打 开 百 度 网
浏 览 一 下 百 度
中 国 新 华 网 打 开
打 开 科 大 讯 飞 网 站
开 启 雅 虎
打 开 杜 鑫 文 的 网 站
进 你 的 官 网
打 开 丰 顺 县 政 府 网
进 入 三 w 点 百 度 点 c o m
进 入 讯 飞 官 网
打 开 网 页 小 米 论 坛
上 百 度 网 站
幻 剑 书 盟
聚 友 网
宜 家
广 西 新 闻 网
打 开 索 尼 官 方 网 站
现 在 打 开 你 的 官 网
打 开 百 度 啊
我 要 打 开 百 度
新 浪 汽 车
九 阴 真 经
我 想 上 网 易
你 帮 我 打 开 网 站
打 开 搜 狐 网 首 页
但 是 我 想 上 腾 讯 网
打 开 互 联 网 啦
上 美 空 网
中 国 新 闻 网 网 站
我 要 上 网 打 开 百 度
