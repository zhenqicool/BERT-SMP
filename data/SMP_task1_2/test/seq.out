0 0 B-datetime_date I-datetime_date 0 0 0 0 0
B-name I-name I-name I-name
0 0 0 0 B-name I-name 0 0 0
0 B-name I-name I-name
B-name I-name I-name
B-name I-name I-name I-name I-name
B-name I-name I-name I-name
0 B-name I-name I-name 0 0
0 B-name I-name I-name I-name
0 0 0 0 0 B-name I-name I-name
0 0 B-name I-name I-name I-name 0 0
B-name I-name
B-name I-name I-name
B-name I-name I-name I-name I-name
0 0 B-name I-name I-name I-name I-name I-name I-name I-name
B-name I-name I-name I-name I-name 0 0
0 0 B-name I-name 0 0
0 0 B-name I-name I-name I-name I-name I-name I-name
0 0 B-name I-name I-name I-name I-name I-name
0 0 0 0 0 0 B-name I-name I-name 0
0 0 0 0 B-name I-name 0
0 0 0 0 B-name I-name
0 0 B-name I-name I-name 0 0
0 0 B-name I-name I-name I-name 0 0
0 0 B-name I-name
0 0 B-name I-name I-name I-name I-name I-name
0 B-name I-name I-name I-name
0 0 B-name I-name I-name I-name I-name I-name
0 0 B-name I-name I-name I-name I-name I-name I-name I-name I-name
0 0 B-name I-name I-name I-name
0 0 0 0 B-name I-name I-name I-name
0 B-name I-name 0 0
B-name I-name I-name I-name
B-name I-name I-name
B-name I-name
B-name I-name I-name I-name I-name
0 0 B-name I-name 0 0 0 0
0 0 0 0 B-name I-name I-name I-name
0 0 B-name I-name 0
0 0 0 0 B-name I-name
B-name I-name I-name I-name
B-name I-name I-name I-name
0 0 0 B-name I-name
你 帮 我 打 开 网 站
0 0 B-name I-name I-name 0 0
0 0 0 0 0 B-name I-name I-name
打 开 互 联 网 啦
0 B-name I-name I-name
B-name I-name I-name I-name I-name 0 0
0 0 0 0 0 0 B-name I-name
